stream-rpc/src/tests/util.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::net::SocketAddr;
+use std::sync::Arc;
+use storage_interface::DbReader;
+use diem_logger;
+use diem_config::config::{RoleType, WebsocketStreamConfig};
+use diem_types::chain_id::ChainId;
+
+use crate::websocket::get_websocket_routes;
+
+fn make_ws_config(fetch_size: u16, queue_size: usize, poll_interval_ms: usize) -> WebsocketStreamConfig {
+    WebsocketStreamConfig {
+        enabled: true,
+        address: "0.0.0.0:7777".parse().unwrap(),
+        content_length_limit: 1000,
+        subscription_fetch_size: fetch_size,
+        send_queue_size: queue_size,
+        poll_interval_ms,
+        tls_cert_path: None,
+        tls_key_path: None,
+    }
+}
+
+#[tokio::test]
+async fn testme() {
+    diem_logger::DiemLogger::init_for_testing();
+    let routes = get_websocket_routes(&Default::default(), Arc::new(()));
+    let mut client = warp::test::ws().handshake(filter).await;
+}
```

Comment:
What is this test supposed to be testing? it doesn't seem to be checking any conditions

Reply:
still heavily WIP here :-)